docs/UserGuide.adoc

```diff
@@ -72,6 +73,11 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Sorting the entire list : `sort`
+
+Sorts all of the contacts in the current addressbook. +
```

Comment:
Maybe you could add that it sorts alphabetically by the first name. Just in case the person viewing the user guide has not read/ has missed line 40.